ui/tests/e2e/search_log.test.ts

```diff
@@ -22,32 +22,62 @@ describe('Search Logs', () => {
       await ppExpect(page).toClick('button#signin_btn')
 
       // jump to search logs page
-      await ppExpect(page).toClick('a#search_logs')
+      await page.waitForSelector('a#search_logs')
+      const searchLogsLink = await page.$('a#search_logs')
+      await searchLogsLink.click()
+
+      // this fails randomly and high possibility, says can't find "a#search_logs" element
+      // await ppExpect(page).toClick('a#search_logs')
 
       // find search form
       const searchForm = await page.waitForSelector('form#search_form')
 
+      // choose time range
+      await ppExpect(searchForm).toClick(
+        'button[data-e2e="timerange-selector"]'
+      )
+      await ppExpect(page).toClick(
+        'div[data-e2e="common-timeranges"] div:last-child'
```

Comment:
what does last-child mean?

Reply:
As its name, it means the element is its parent's last child.

![image](https://user-images.githubusercontent.com/1284531/85923678-c913be00-b8bf-11ea-847b-5e437bfc59a4.png)

Reply:
Ok, I got it. Maybe it's better to modify our time range picker and provide some better ids :) It will better adapt to structural or style changes.

Reply:
Add ``data-e2e={`timerange-${seconds}`}`` to each item.